website/__tests__/redux/integration/ElementIntegrationTests.jsx

```diff
@@ -24,35 +26,42 @@ describe('PositionsIntegrationTests.', () => {
     testStore = null;
   });
 
-  test('initPositions_Dispatch', () => {
+  test('initElements_Dispatch', () => {
     const elemList = {
       testingANewItem: {
-        position: { x: 1000, y: 33 }
+        position: { x: 1000, y: 33 },
+        size: {width: 100, height: 100 },
```

Comment:
You're missing a space in between { and w for the record if you want to fix it for a future push lol.